ui/shared/auth/Login.jsx

```diff
@@ -105,45 +112,40 @@ class Login extends React.Component {
     const { user } = this.props;
 
     return (
-      <React.Fragment>
+      <span id="auth-menu">
         {user.isLoggedIn && (
-          <span className="dropdown">
-            <button
+          <UncontrolledButtonDropdown>
+            <DropdownToggle
+              caret
               id="logoutLabel"
-              type="button"
               title={`Logged in as: ${user.email}`}
-              data-toggle="dropdown"
-              className="btn btn-view-nav"
+              className="btn-view-nav"
             >
-              <div className="dropdown-toggle">
-                <div className="nav-user-icon">
-                  <span className="fa fa-user pull-left" />
-                </div>
-                <div className="nav-user-name">
-                  <span>{user.fullName}</span>
-                </div>
+              <div className="nav-user-icon">
+                <span className="fa fa-user pull-left" />
               </div>
-            </button>
-            <ul
-              className="dropdown-menu nav-dropdown-menu-right"
-              role="menu"
-              aria-labelledby="logoutLabel"
-            >
-              <li>
-                <a onClick={this.logout} className="dropdown-item">
-                  Logout
-                </a>
-              </li>
-            </ul>
-          </span>
+              <div className="nav-user-name">
+                <span>{user.fullName}</span>
+              </div>
+            </DropdownToggle>
+            <DropdownMenu aria-labelledby="logoutLabel">
+              <DropdownItem onClick={this.logout}>
+                <span className="dropdown-item">
```

Comment:
this span with the ``dropdown-item`` class is not necessary anymore.  I think you can just remove it and leave the inner ``span``